### PR TITLE
injecting prettydiff2.x into Atom Beautify

### LINKED ISSUE #881: Pretty Diff v2 and NPM
Pretty Diff v2 is about 1-2 weeks away.  V2 will not be published to NPM though :(

For more information please see:
- https://github.com/prettydiff/prettydiff/issues/291
- https://github.com/prettydiff/prettydiff/issues/292

Pretty Diff will continue to publish to Bower, Github, and http://prettydiff.com and additional package managers are being investigated.  Due to time limitations between this enhancement, my two jobs, and preparation for attendance at [WOCS](https://en.wikipedia.org/wiki/Warrant_Officer_Candidate_School_%28United_States_Army%29) v2 will be released before additional publication and package management options are supported.

### PR DESCRIPTION
**Pull request by prettydiff**
### What does this implement/fix? Explain your changes.
Brings Pretty Diff updates to Atom Beautify
...

### Does this close any currently open issues?
fixes #881
...

### Any other comments?
Pretty Diff is directly embedded.  It does not require any second or third party distribution.  The way to update Pretty Diff is to install over it from http://prettydiff.com/downloads/prettydiff/prettydiff_cli_latest.zip using biddle or manually.

When biddle version 2 is released the source will change from the Pretty Diff domain URI to a IPFS hash managed through the Pretty Diff domain.
...

### Checklist

Check all those that are applicable and complete.

- [ ] Merged with latest `master` branch
- [ ] Regenerate documentation with `npm run docs`
      * I did this, but seemed to generate faulty output by removing the final curly brace from package.json and changing the default value of YAML padding property, in options.json, to null
- [x] Add change details to `CHANGELOG.md` under "Next" section
- [ ] Added examples for testing to [examples/ directory](examples/)
- [ ] Travis CI passes (Mac support)
- [ ] AppVeyor passes (Windows support)

### CODE REVIEW

**Comment by prettydiff** [comment]
This pull request just puts Pretty Diff into Atom Beautify as a dependency without distribution (directly embedded).  biddle is not included here, as that is not how biddle is designed to work, but it can be added as an additional dependency for checking package status.

**Comment by prettydiff** [comment]
Seems to be breaking the build.  This could be due to a change in the API.

old - `result = prettydiff.api(options)[0]`
new - `result = prettydiff(options)`

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
 .DS_Store
 npm-debug.log
-node_modules
+node_modules/*
+!node_modules/prettydiff
```
Comment (Glavin001) [inline .gitignore]: I do not want to commit to the Git repository these files.

**Comment by Glavin001** [comment]
Cleaning up PRs. Can be reopened once there are changes to review. Thanks!

**Pull request closed without merging**